bundles/SyliusReviewBundle/forms.rst

```diff
@@ -0,0 +1,57 @@
+Forms
+=====
+
+The bundle ships with a default review type, that should be inherited by your application's specific review type.
```

Comment:
*The bundle ships with a default review type* Is quite weired sentence. Did you mean *The bundle is shiped with a default review type* or *The bundle ships a default review type*